packages/coding-standards/shopsys-coding-standard.yml

```diff
@@ -0,0 +1,215 @@
+    PhpCsFixer\Fixer\ClassNotation\SelfAccessorFixer: ~
+    PhpCsFixer\Fixer\Semicolon\SemicolonAfterInstructionFixer: ~
+    PhpCsFixer\Fixer\CastNotation\ShortScalarCastFixer: ~
+    PhpCsFixer\Fixer\NamespaceNotation\SingleBlankLineBeforeNamespaceFixer: ~
+    PhpCsFixer\Fixer\Semicolon\SpaceAfterSemicolonFixer: ~
+    PhpCsFixer\Fixer\StringNotation\SingleQuoteFixer: ~
+    PhpCsFixer\Fixer\Comment\SingleLineCommentStyleFixer:
+        comment_types:
+            - hash
+    PhpCsFixer\Fixer\Operator\StandardizeNotEqualsFixer: ~
+    PhpCsFixer\Fixer\Strict\StrictParamFixer: ~
+    PhpCsFixer\Fixer\Operator\TernaryOperatorSpacesFixer: ~
+    PhpCsFixer\Fixer\ArrayNotation\TrailingCommaInMultilineArrayFixer: ~
+    PhpCsFixer\Fixer\ArrayNotation\TrimArraySpacesFixer: ~
+    PhpCsFixer\Fixer\Operator\UnaryOperatorSpacesFixer: ~
+    PhpCsFixer\Fixer\ArrayNotation\WhitespaceAfterCommaInArrayFixer: ~
+
+    # Code Metrics
+    ObjectCalisthenics\Sniffs\Files\ClassTraitAndInterfaceLengthSniff:
+        maxLength: 550
+    ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        maxLength: 60
+    ObjectCalisthenics\Sniffs\Metrics\PropertyPerClassLimitSniff:
+        maxCount: 30
+
+parameters:
+    exclude_files:
+        - '*tests/ShopBundle/Test/Codeception/_generated/AcceptanceTesterActions.php*'
+    skip:
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # it is not necessary to have blank line after control structure
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ControlStructureSpacingSniff.NoLineAfterClose: ~
+        # allow empty "catch (Exception $exception) { }"
+        PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedCatch: ~
+
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # it is not necessary to have blank line after control structure
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\WhiteSpace\ControlStructureSpacingSniff.NoLineAfterClose: ~
+        # allow empty "catch (Exception $exception) { }"
+        PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\EmptyStatementSniff.DetectedCatch: ~
+
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+        # skip known issues for framework
+            - packages/framework/src/DataFixtures/Base/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/Demo/OrderDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/MailTemplateDataFixture.php
+            - packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php
```

Comment:
This can be actually simplified by using fnmatch() format, see docs: https://github.com/symplify/easyCodingStandard/#ignore-what-you-cant-fix

```diff
 parameters: 
     skip:
         ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
             # skip known issues for framework
-            - packages/framework/src/DataFixtures/Base/MailTemplateDataFixture.php
-            - packages/framework/src/DataFixtures/Demo/CategoryDataFixture.php
-            - packages/framework/src/DataFixtures/Demo/OrderDataFixture.php
-            - packages/framework/src/DataFixtures/DemoMultidomain/CategoryDataFixture.php
-            - packages/framework/src/DataFixtures/DemoMultidomain/MailTemplateDataFixture.php
+            - "*packages/framework/src/DataFixtures/*/*DataFixture.php"
```